docs/reference/evercrypt/mac/index.md

```diff
@@ -30,14 +30,14 @@ The available hash functions.
 
 The allowed values are ...
 
-* `Spec_Hash_Definitions_SHA2_224`,
-* `Spec_Hash_Definitions_SHA2_256`,
-* `Spec_Hash_Definitions_SHA2_384`,
-* `Spec_Hash_Definitions_SHA2_512`,
-* `Spec_Hash_Definitions_SHA1`,
-* `Spec_Hash_Definitions_MD5`,
-* `Spec_Hash_Definitions_Blake2S`, and
-* `Spec_Hash_Definitions_Blake2B`.
+* `Spec_Hash_Definitions_SHA2_224` (KEY_LEN=64, DIGEST_LEN=28),
```

Comment:
What's `key_len` for the hash functions here?

Reply:
Yeah... I should make that clearer. This value is used below. It is the recommended key size, i.e., one that neither needs padding nor hashing. We should introduce that right before using it. Would that suffice?

Reply:
Yeah, add some of the information that you removed here (because other values are possible and might be more appropriate depending on the use case).
Also it needs to be clear that you talk about the key used for HMAC, not the one that you could use with blake2 for example.